### ISSUE TITLE
Is there a tts demo for softmax wavenet vocoder

### ISSUE THREAD

**Issue by everange-ustc** [espnet/espnet#2298]
The pretrained model of the softmax wavenet vocoder is provided, but the demo (notebook) does not include the wavenet. Is there any possible to add this part in the notebook?

**Comment by kan-bayashi** [comment]
Sorry, we do not support wavenet in colab demo, since it requires very long time to generate.
There is a bash script to run the decoding with wavenet.
The following part might help you. 
https://github.com/espnet/espnet/blob/b4413f6259c49d2543db1e10417c08118a09d990/utils/synth_wav.sh#L342-L355
